.github/workflows/ai-translate.yml

```diff
@@ -6,10 +6,21 @@ on:
       - main
   workflow_dispatch:
 
+permissions:
+  contents: read # for checkout
+
 jobs:
   translate-and-pr:
+    permissions:
+      contents: write # for push
```

Comment:
```suggestion
```
This permission config only applies to the `GITHUB_TOKEN`, generated by GitHub. Since we're using ecospark to generate the token instead we should be able to omit this permission escalation and leave the token at just read permissions 🤔